Source/Core/Core/NetPlayClient.h

```diff
@@ -155,6 +162,9 @@ class NetPlayClient : public TraversalClientClient
   void SendStartGamePacket();
   void SendStopGamePacket();
 
+  void SyncSaveDataResponse(const bool success);
```

Comment:
`const` can be omitted here in the prototype. It's fine to leave in the definition, however.